packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -312,29 +326,25 @@ const Multiselect = React.forwardRef<HTMLDivElement, IMultiselectProps & ThemePr
                 placeholder: selectedItems.length === 0 ? placeholder : undefined
               }) as any)}
             />
```

Comment:
From #607 we learned that there should be a child `<div>` (surrounding the tags) that prevents tags from overflowing into the container padding provided by the `StyledMultiSelect`

Reply:
I've added a wrapping `<div>` to help with this styling. Continuing the work in https://github.com/zendeskgarden/react-components/pull/610#discussion_r376608394